src/spectrecoff-cli/commands/Status.fs

```diff
@@ -40,4 +40,28 @@ type StatusExample() =
 
         (startWithCustomSpinner normalThinkingSpinner thinkingProcess).Wait()
         P "No result" |> toConsole
+
+        let asyncProcess (context: StatusContext) =
+            async {
+                do! Async.Sleep 3000
```

Comment:
For clarity, I'd define the two spinners at the top, then write the 'flow':

```fs
let harderThinkingSpinner = // ...
let maximumThinkingSpinner = // ...

do! Async.Sleep 3000
updateWithCustomSpinner harderThinkingSpinner context |> ignore

do! Async.Sleep 3000
updateWithCustomSpinner maximumThinkingSpinner context |> ignore

do! Async.Sleep 3000
return 42
```

Reply:
Good thinking (hehe)